modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableRowSkin.java

```diff
@@ -185,6 +189,7 @@ private void setupTreeTableViewListeners() {
                 }
                 return null;
             }
+            // fall through
```

Comment:
There is no "fall through" here as we return from all conditions in previous case.

Reply:
same comment, there is a fall through.

Reply:
No, there isn't.

Reply:
this is embarrassing - you are right.  the only fall through case is on line 169, but it's clearly wrong, as it should not be falling through to CELL_AT_ROW_COLUMN.

in the case of a null selection model, should queryAccessibleAttribute(SELECTED_ITEMS) return null or an empty list?